OpenAI-DotNet/ThreadMessages/CreateThreadMessageRequest.cs

```diff
@@ -0,0 +1,48 @@
+using System.Collections.Generic;
+using System.Text.Json.Serialization;
+using OpenAI.Chat;
+
+namespace OpenAI.ThreadMessages
+{
+    public class CreateThreadMessageRequest
```

Comment:
Seems like a duplicate class from Threads.

I think this whole namespace should be combined since it's technically the same root endpoint.